### DISCUSSION TITLE
Password change

### DISCUSSION THREAD

**Discussion by maverusfxt** [codefori/vscode-ibmi#458] [Q&A] [answered]
Thanks for this extension to VS Code! VS Code is my editor of choice.

How to I change/re-enter my password in the extension (I have changed my IBM i password)? I have done it before but cannot find it now. No info on the documentation page.

If it is in Login Settings, that won't display when I click to open it.

![image](https://user-images.githubusercontent.com/30267469/148092410-f125e7d1-938a-4499-a3ce-3e9f7be9d9c0.png)

**Comment by worksofliam** [comment]
Hey @maverusfxt!

Before connecting, right-click on your connection and select Login settings. From there a Window will popup where you can change your password.

**Comment by maverusfxt** [comment]
Thanks for the reply. I have tried to access the login settings before and after connecting. I get an error when trying to access the setting before connecting.

![image](https://user-images.githubusercontent.com/30267469/148098891-05d63a74-dd96-47a8-b27a-916c89e2dc92.png)

**Reply by worksofliam** [reply]
@maverusfxt Me too! Let me look into that.

**Comment by worksofliam** [comment, marked as answer]
@maverusfxt I would recommend creating a new connection in the mean time. There will be a fix out this afternoon for that!

**Comment by maverusfxt** [comment]
Thanks for the quick response!
